packages/office-ui-fabric-react/src/components/DocumentCard/__snapshots__/DocumentCardTitle.test.tsx.snap

```diff
@@ -0,0 +1,10 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`DocumentCardTitle shows title with secondary title style 1`] = `
```

Comment:
Not sure if this snapshot is seeing the difference in between showAsSecondaryTitle set to true or false.
If the intention was to check for color difference than maybe a VR story would be more appropriate.

Reply:
You're totally right, good call. I'll write a VR test instead.